mainwindow.cpp

```diff
@@ -92,12 +90,10 @@ MainWindow::MainWindow(QSettings &settings)
 
     // We connect all widgets for option with radio buttons to keep
     // enabled only widgets of selected mode.
-    // TODO: Easier way? Maybe make connections in designer?
```

Comment:
To me it's the opposite, I hate it when connections are made in the designer. I prefer to CTRL+FIND the signals I'm looking for and modify it in the code. Might be personal choice